kerlescan/audit_logging.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import json
+import base64
+
+from kerlescan.service_interface import get_key_from_headers
+
+AUDIT_LEVEL_NAME = "AUDIT"
+AUDIT_LEVEL_NUM = int((logging.INFO + logging.WARNING) / 2)
+
+
+def audit(self, message, *args, **kws):
```

Comment:
Can we add a comment describing what kws args that can be passed and their expected values?